ns1/rest/zones.py

```diff
@@ -18,16 +19,22 @@ class Zones(resource.BaseResource):
         "secondary",
         "hostmaster",
         "meta",
+        "name",
         "networks",
         "link",
+        "views",
+        "zone",
     ]
     BOOL_FIELDS = ["dnssec"]
 
-    def _buildBody(self, zone, **kwargs):
-        body = {}
-        body["zone"] = zone
+    def _buildBody(self, z, **kwargs):
+        if "name" in kwargs and kwargs["name"] != "zone":
```

Comment:
`"zone"` should be `z` I think?

Reply:
yeah ouch, that's a dramatically bad typo